src/App.vue

```diff
@@ -1,62 +1,14 @@
 <template>
   <div id="app">
-    <header class="header">
-      <ul class="btnArea">
-        <li><button class="btn" @click="changeMonth(-1)">&lt;</button></li>
-        <li><button class="btn" @click="currentMonth">当月</button></li>
-        <li><button class="btn" @click="changeMonth(1)">&gt;</button></li>
-      </ul>
-      <div>{{ formatMonth(activeDate) }}</div>
-      <button class="btn" @click="openModal">登録</button>
-      <!-- TODO: 登録ボタンの実装 -->
-      <vue-modal-2
-        name="scheduleRegister-modal"
-        :headerOptions="{
-          title: '予定を登録',
-        }"
-        :footerOptions="{
-          btn1: 'キャンセル',
-          btn2: '登録',
-          btn2Style: {
-            backgroundColor: 'green',
-          },
-          btn2OnClick: () => {
-            doDelete();
-          },
-          btn1OnClick: () => {
-            $vm2.close('scheduleRegister-modal');
-          },
-        }"
-        @on-close="closeModal"
-        class="scheduleRegister-modal"
-      >
-        <div class="scheduleRegister-content">
-          <div>
-            <InputText v-model="inputText" />
-          </div>
-          <div><InputDate v-model="inputDate" /></div>
-          <div class="scheduleRegister-startTime">
-            <span>開始時間</span><InputTime v-model="inputTime" />
-          </div>
-          <div class="scheduleRegister-endTime">
-            <span>終了時間</span>
-            <InputTime v-model="inputTime" />
-          </div>
-        </div>
-      </vue-modal-2>
-    </header>
+    <CalenderHeader
+      :active-date="activeDate"
+      @changeMonth="changeMonth"
+      @currentMonth="currentMonth"
```

Comment:
ここはイベントなので、
```suggestion
      @onChange="changeCurrentMonth"
```
こんな感じの方が良いかと。

Reply:
修正しました。